src/main/java/com/sericulture/authentication/service/UserInfoDetails.java

```diff
@@ -1,20 +1,23 @@
 package com.sericulture.authentication.service;
 
+import com.sericulture.authentication.model.entity.RoleMaster;
 import com.sericulture.authentication.model.entity.UserInfo;
 import lombok.Getter;
 import lombok.Setter;
 import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
 import org.springframework.security.core.userdetails.UserDetails;
 
 import java.util.Collection;
 import java.util.List;
+import java.util.Objects;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.Objects' import. Should be before 'org.springframework.security.core.userdetails.UserDetails'.